crossbar/worker/test/test_router.py

```diff
@@ -257,3 +267,194 @@ def test_start_router_component_wrong_baseclass(self):
             str(e.exception.args[0]))
 
         self.assertEqual(len(r.get_router_components()), 0)
+
+
+class WSGITests(TestCase):
+
+    skip = WSGI_TESTS
+
```

Comment:
This is a nicer way to skip test-suites than the ``if something: ...all the code...`` idiom :)